agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -446,6 +450,8 @@ public static class LegacyRequestIdPropagation {
   public static class InternalConfiguration {
     // This is used for collecting internal stats
     public Statsbeat statsbeat = new Statsbeat();
+    public PreAggregatedStandardMetricsConfiguration preAggregatedStandardMetric =
```

Comment:
(lowercase "a" and trailing "s")
```suggestion
    public PreAggregatedStandardMetricsConfiguration preaggregatedStandardMetrics =
```